File: cursos/php_poo/exercicio/aula01/Caneta.php
<?php 

class Caneta{
    var $modelo;
    var $cor;
    var $ponta;
    var $carga;
    var $tampada;

    function rabiscar(){

        if($this->tampada == true):
            echo "<p>Não posso rabiscar...</p>";

        else:
        echo "<p>Estou rabiscando!</p>";
        
        endif;
    }

    function tampar(){

        $this->tampada = true;

    }

    function destampar(){

        $this->tampada = false;
    }
}

File: cursos/java_poo/Fundamentos/Aula 03/src/Caneta.java

public class Caneta {
	
	public String modelo;
	public String cor;
	private float ponta;
	protected int carga;
	private boolean tampada;
	
	public void status() {
		System.out.println("Uma caneta modelo " + this.modelo);
		System.out.println("cor " + this.cor);
		System.out.println("ponta " + this.ponta);
		System.out.println("Carga "+ this.carga + "%");
		System.out.println("Está tampada? " + this.tampada);
	}
	public void rabiscar() {
		if(this.tampada == true) {
			System.out.println("A caneta está tampada, Não posso rabiscar");
		}
		else {
			System.out.println("Estou rabiscando");
		}
	}
	protected void tampar() {
		this.tampada = true;
	}
	public void destampar() {		
		this.tampada = false;		
	}
}
